### DISCUSSION TITLE
Can you add Sky Sports and Ten sport and ESPN

### DISCUSSION THREAD

**Discussion by d0tb0t71** [iptv-org/community#985] [Q&A]
There is no available channel for Sky Sport , Ten Sport , ESPN. Please Add them

**Comment by SpiralBrad** [comment]
@d0tb0t71 If you set up streams for them, I'm sure the team will be happy to add them.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
